projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/handlers.py

```diff
@@ -95,6 +95,29 @@ def post(self):
             self.finish(json.dumps({"message": f"{e}", "error": "true"}))
 
 
+class TransformJobHandler(APIHandler):
+    """
+    Class responsible for handling POST request for transforming a Data Job given its name, team,
+    Rest API URL, and the path to the data job's directory
+    Response: return a json formatted str including:
+        ::error field with error message if an error exists
+        ::message field with status of the Vdk operation
+    """
+
+    @tornado.web.authenticated
+    def post(self):
+        input_data = self.get_json_body()
+        try:
+            status = VdkUI.transform_job(
```

Comment:
here the code should be removed
The person who is adding the implementation for the operation will add those I think it is better to remove it
 I think you can just leave the place blank or to return sth like "Successfully connected to handler" for now

Reply:
Done.